evalml/tests/utils_tests/test_logger.py

```diff
@@ -108,7 +108,13 @@ def test_pipeline_count(
     else:
         X, y = X_y_regression
     if not allowed_families:
-        _ = AutoMLSearch(X_train=X, y_train=y, problem_type=type_, verbose=verbose)
+        _ = AutoMLSearch(
```

Comment:
This tests pipeline count for iterative but this should be generalized.

Reply:
@bchen1116  The test itself would be generalized in #2868 (its included in the list of tests) and if you're referring to the `# of pipelines ready to search` output, I put up #3283 to track that!